sub-projects/hub/src/app/components/auth/login-page.jsx

```diff
@@ -0,0 +1,112 @@
+import React from 'react';
```

Comment:
I think this file is unnecessary

Reply:
Yeah I guess it creeped into my code during the merge 😆